electron/localDB.ts

```diff
@@ -168,6 +168,7 @@ const intialiseUserTable = (db) => {
         'CREATE TABLE users (\
         username TEXT PRIMARY KEY,\
         password TEXT NOT NULL,\
+        name TEXT,\
```

Comment:
What is the difference between `name` and `username`?

Reply:
> What is the difference between `name` and `username`?

The username is the unique ID for logging in, name is combination of first name and last name. otherwise, I can't quite remember, it was giving an error.